### DISCUSSION TITLE
Youtube app or broswer not blocking ads via Adguard Home why?

### DISCUSSION THREAD

**Discussion by baspax1** [AdguardTeam/AdGuardHome#6186] [Q&A]
Seems to blocking https websites ads, but when i open youtube.com i get ads on videos, is there any way to block such as ublock ?

**Comment by iam-py-test** [comment]
YouTube ads can not be blocked by AdGuard Home without breaking YouTube, as AdGuard Home can only block entire domains, and (I think) YT ads are not all served from a designated domain.
Thanks.

**Reply by baspax1** [reply]
It's a shame, I thought that with AdGuard Home I would solve the problem of ads in general, but I'm just fixing a part of them...

**Reply by iam-py-test** [reply]
That is just a limitation of blocking ads at the DNS level. AdGuard Home is great because it can block ads from all apps on pretty much all devices. However, it can only block entire domains/subdomains.
The AdGuard browser extension and uBlock Origin have far more power, but only can block ads in your browser.
Thanks
